create-featurenet/action.yml

```diff
@@ -55,6 +55,10 @@ inputs:
     description: 'Wait for heads finalization'
     required: false
     default: "false"
+  sudo-account-id:
+    description: 'Sudo account ID'
+    required: true
```

Comment:
Why is there `required: true` if it's got a default value?

Reply:
good point, it should be false since otherwise it's a API breaking change